### ISSUE TITLE
Socket activity but no reload triggered

### ISSUE THREAD

**Issue by gilbox** [gaearon/react-hot-loader#74]
On page load I see:

       [HMR] Waiting for update signal from WDS...

Then, when I save a component, webpack recompiles and I see some frames for the WebSocket (screenshot below), but there is no [HMR] activity. 

I tried setting a breakpoint in only-dev-server.js at the first line of the `"message"` event handler, but it never breaks.

Any ideas?

![image](https://cloud.githubusercontent.com/assets/2136203/6122974/758f50a2-b0c1-11e4-94b4-1bb27202aeae.png)

**Comment by gaearon** [comment]
What I usually do is compare every little configuration piece with [react-hot-boilerplate](https://github.com/gaearon/react-hot-boilerplate) until I find something that differs. Can you try doing that?

**Comment by gilbox** [comment]
Thanks a lot for the push. I was trying to get this working with rails. I had been looking at react-hot-boilerplate, but on your suggestion I went back and got it working with just `server.js` (without rails), and then changed the paths to be more rails-like (`/app/assets/javacripts`), and changed the port to `8080`. 

Then I changed just one value, `publicPath`, and it worked with rails:

`/app/assets/javascripts/` => `http://localhost:8080/app/assets/javascripts/`
